LiskHQ/lisk-hub
# Dates must be translated in selected language - Closes #2354

### What issue have I solved?

#2354 

### How have I implemented/fixed it?

#### Using Moment
- [x] Transaction details page.
- [x] Wallet page, transactions list.
- [x] Other wallets, transactions list.
- [x] Wallet, Balance chart
- [x] Other wallet, balance chart
- [x] Delegate profile
- [x] Calendar datepicker on transactions filter

#### Needs translation on common.json
- [x] Dashboard -> Community feed
- [x] Transactions list -> to show [today and yesterday] correctly.

I updates strings that were missing the `t()` function, and updated places using moment.js to consider the selected language  
Moment.js already has localization, only issue is with the calendar function, used to generate string as `[Today] DD.MM.YYYY`, that should be translated, and also the date formats updated if needed.

### How has this been tested?

It's needed to set a `feature-flag-language` into localStorage to see the option in settings. 
```js
localStorage.setItem('feature-flag-language', true);
```

1. Go to Settings;
2. Change language;
3. See that dates are shown with German month names.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
